decidim-admin/app/assets/javascripts/decidim/admin/sort_steps.js.es6

```diff
@@ -6,6 +6,13 @@
 // should have a `data-sort-url` attribute with the URL where the data should
 // be posted to.
 $(() => {
+ sortSteps();
+});
+$( document ).on('turbolinks:load' , function() {
+  sortSteps();
+});
+
+function sortSteps(){
   const sortableElement = $('#steps tbody');
 
   if (sortableElement[0]) {
```

Comment:
It's better if you just check if `sortableElement.length > 0` but both solutions are good :+1: